secboot/secboot_tpm.go

```diff
@@ -338,10 +372,12 @@ func SealKey(key EncryptionKey, params *SealKeyParams) error {
 
 	// Seal key to the TPM
 	creationParams := sb.KeyCreationParams{
-		PCRProfile: pcrProfile,
-		PINHandle:  pinHandle,
+		PCRProfile:             pcrProfile,
+		PCRPolicyCounterHandle: policyCounterHandle,
 	}
-	return sbSealKeyToTPM(tpm, key[:], params.KeyFile, params.TPMPolicyUpdateDataFile, &creationParams)
+	_, err = sbSealKeyToTPM(tpm, key[:], params.KeyFile, &creationParams)
```

Comment:
we likely need to capture the auth key here as well to store in a file anyway for the recovery case anyway